sidekick_core/lib/src/commands/format_command.dart

```diff
@@ -96,17 +103,18 @@ class FormatCommand extends Command {
 
     final allFiles =
         SidekickContext.projectRoot.findFilesToFormat(globExcludes).toList();
+    final withoutGenerated = filterGeneratedFiles(allFiles).toList();
```

Comment:
Is this filename still correct?
If formatGenerated is true, this includes all files.

Reply:
Any better idea than `maybeWithithOrWithoutGeneratedFiles`?

Reply:
filteredFiles?